src/modules/comment.test.js

```diff
@@ -0,0 +1,29 @@
+/** @jest-environment jsdom */
+/* eslint-disable global-require */
```

Comment:
- [ ] Same here, Please make sure you fix all linter errors instead of disabling them, A linter is great for identifying errors when you use standard rules. Remember, a linter analyzes your code for stylistic and programming errors against the rules it knows. 💯 😉

Reply:
The error has been resolved here aswell